src/common.speech/SpeechSynthesisConnectionFactory.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import {
+    ProxyInfo,
+    WebsocketConnection, WebsocketMessageAdapter,
+} from "../common.browser/Exports";
+import {
+    IConnection,
+    IStringDictionary
+} from "../common/Exports";
+import { PropertyId } from "../sdk/Exports";
+import {
+    AuthInfo,
+    SynthesizerConfig,
+    WebsocketMessageFormatter
+} from "./Exports";
+import { ISynthesisConnectionFactory } from "./ISynthesisConnectionFactory";
+import {
+    QueryParameterNames
+} from "./QueryParameterNames";
+
+export class SpeechSynthesisConnectionFactory implements ISynthesisConnectionFactory {
+
+    private readonly synthesisUri: string = "/cognitiveservices/websocket/v1";
+
+    public create = (
+        config: SynthesizerConfig,
+        authInfo: AuthInfo,
+        connectionId?: string): IConnection => {
+
+        let endpoint: string = config.parameters.getProperty(PropertyId.SpeechServiceConnection_Endpoint, undefined);
+        const region: string = config.parameters.getProperty(PropertyId.SpeechServiceConnection_Region, undefined);
+        const hostSuffix = region.startsWith("china") ? ".azure.cn" : ".microsoft.com";
```

Comment:
>hostSuffix = region.startsWith("china") ? ".azure.cn" : ".microsoft.com"; [](start = 14, length = 73)

I think the rest of the factories also need this change.

Reply:
I will update this change to other factories.

Reply:
will do it in another PR, resolving conversation here.

Reply:
see #163